Packs/VMRay/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "VMRay Analyzer",
     "description": "Analyze files and URLs using the VMRay Platform for accurate threat intelligence and high-quality IOCs.",
     "support": "partner",
-    "currentVersion": "1.1.12",
+    "currentVersion": "1.2.0",
```

Comment:
I think a minor update is enough.
What do you think?
https://xsoar.pan.dev/docs/documentation/release-notes

Reply:
Hey,
don't understand your note. You made it a "bigger" version.

Reply:
I mean it could be like this
`"currentVersion": "1.1.13"` insteadof `"currentVersion": "1.2.0"`.

Reply:
ah sure should be sorry. 
Year absolutly right. doing several things at the same time, sorry.

Reply:
@MosheEichler I changed the version as requested